cesium/internal/unary/iterator.go

```diff
@@ -227,7 +241,7 @@ func (i *Iterator) Len() (l int64) {
 	return
 }
 
-func (i *Iterator) Error() error { return i.err }
+func (i *Iterator) Error() error { return core.NewErrorWrapper(i.Channel)(i.err) }
```

Comment:
Kinda weird but I guess its ok

Reply:
I thought since the error wrapper was only used in two places, it did not make sense to store it as as field in the iterator struct